website/dbt-versions.js

```diff
@@ -0,0 +1,31 @@
+exports.versions = [
+  {
+    version: "1.2",
+    EOLDate: "2023-01-01"
+  },
+  {
+    version: "1.0",
+    EOLDate: "2022-05-20"
+  },
+  {
+    version: "0.21",
```

Comment:
@runleonarun put real life EOL scenarios in here. 

cc @jtcohen6 for awareness